src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceComplaintsAsserts.java

```diff
@@ -122,5 +123,26 @@ public void verifyCriteriaWasAssociatedToComplaint(final String criteria) {
     public void iSeeComplaintsReportingPageTitle(final String expectedPageTitle) {
         assertTrue(getDriver().getTitle().contains(expectedPageTitle));
     }
+    
+    @Then("^the complaint with given ONC-ACB Complaint ID \"([^\"]*)\" and Status \"([^\"]*)\" is displayed in complaints view table$")
+    public void complaint_with_given_ONC_ACB_Complaint_ID_and_Status_should_be_added_to_CHPL(String acbComplaintId, String status) {
+        getDriver().navigate().refresh();
+        boolean isFound = false;
+
+        WebElement table = SurveillanceComplaintsPage.complaintsTable(getDriver());
+        List<WebElement> rows = table.findElements(By.xpath("tbody/tr"));
 
+        for (WebElement row : rows) {
+            List<WebElement> cols = row.findElements(By.xpath("td"));
+            if (cols.get(ACBCOMPLAINTID_COL).getText().equals(acbComplaintId)) {
+                isFound = true;
+                break;
```

Comment:
The "Then" here says that the complaint with ID <something> AND status <something> is found, but if the code gets to the point, where it finds the ID, it will then return true and break out of the loop, but it has not checked that the status matches <something>